aerogear/mobile-security-service-operator
# Remove role and role binding since has no need for it.

## Motivation
https://issues.jboss.org/browse/AEROGEAR-8968

## What
- Remove the role and role_binding and the commands which were using it
- Update readme

## Why
It is not a requirement has no need for we to spend time on it. 

## Verification Steps
 
Use this PR to install the operator and ensure that after run "make create-all" all will be working fine.

## Checklist:


## Progress
